lib/src/main/resources/dcx/2020/03/dcx-dai.xsd

```diff
@@ -0,0 +1,334 @@
+                Use of a urified DAI (starting with info:eu-repo/dai/nl/) is allowed.
+            </xs:documentation>
+        </xs:annotation>
+        <xs:restriction base="xs:string">
+            <xs:pattern value="(info:eu-repo/dai/nl/){0,1}([0-9]{8,9})([0-9xX]{1})"/>
+        </xs:restriction>
+    </xs:simpleType>
+
+    <!-- =================================================================================== -->
+    <xs:complexType name="ORCIDType">
+        <xs:complexContent>
+            <xs:extension base="dc:SimpleLiteral">
+                <xs:attribute name="ORCID" type="dcx-dai:SimpleORCIDType" use="required"/>
+                <xs:attribute ref="xml:lang" use="prohibited"/>
+            </xs:extension>
+        </xs:complexContent>
+    </xs:complexType>
+
+    <!-- =================================================================================== -->
+    <xs:simpleType name="SimpleORCIDType">
+        <xs:annotation>
+            <xs:documentation xml:lang="en">Use of a urified ORCID (starting with https://orcid.org/) is recommended.
+            Possible values:
+                0000-0002-1825-0097
+                https://orcid.org/0000-0001-5109-3700
+                https://orcid.org/0000-0002-1694-233X
+            </xs:documentation>
+        </xs:annotation>
+        <xs:restriction base="xs:string">
+            <xs:pattern value="(https://orcid.org/){0,1}([0-9]{4}-){3}[0-9]{3}[0-9xX]{0,1}"/>
+        </xs:restriction>
+    </xs:simpleType>
+
+    <!-- =================================================================================== -->
+    <xs:complexType name="ISNIType">
+        <xs:complexContent>
+            <xs:extension base="dc:SimpleLiteral">
+                <xs:attribute name="ORCID" type="dcx-dai:SimpleORCIDType" use="required"/>
+                <xs:attribute ref="xml:lang" use="prohibited"/>
+            </xs:extension>
+        </xs:complexContent>
+    </xs:complexType>
+
+    <!-- =================================================================================== -->
+    <xs:simpleType name="SimpleISNIType">
+        <xs:annotation>
+            <xs:documentation xml:lang="en">Use of a urified ISNI (starting with http://isni.org/isni) is recommended.
+            Possible values:
+                0000 0001 2281 955X
+                http://isni.org/isni/000000012281955X
+                ISNI:000000012281955X
+            </xs:documentation>
+        </xs:annotation>
+        <xs:restriction base="xs:string">
+            <xs:pattern value="(http://isni.org/isni/[0-9]{15}[0-9X])|((ISNI:? ?)?([0-9]{4} ?){3}[0-9]{3}[0-9X])"/>
+            <!--
+            No lower case X (as in previous implementation)
+            No spaces in case of an URL, an URL 'prefix' is lower case only
+            If not an URL
+            * An optional upper case prefix 'ISNI' followed an optional semicolon and/or space.
+            * No other separators than spaces, Groups of 8 or 12 digits will also pass
```

Comment:
```suggestion
            No lower case 'x' (as in previous implementation)
            No spaces in case of a URL; a URL 'prefix' is lowercase only
            If not a URL:
            * An optional uppercase prefix 'ISNI' followed by an optional semicolon and/or space
            * No other separators than spaces; groups of 8 or 12 digits are also allowed
```